docs/contributing/how-to-contribute-to-docs.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Contribute to docs
+type: docs
+menu: contributing
+---
+
+# How to contribute to Docs/Website
```

Comment:
nit: Maybe I'd add link to the https://github.com/improbable-eng/thanos/tree/master/docs folder directly so user knows where to look for the files

Reply:
Very true